mobify/progressive-web-scaffold
# Basic checkout containers

The very basic start of Shipping, Payment, and Confirmation Checkout pages.

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1015, https://mobify.atlassian.net/browse/WEB-1014

## Changes
- Added the basic containers for the Checkout flow
- Disabled page fetches for these pages until we figure out what the long term plan is
- Added links between the pages

## How to test-drive this PR
- Preview and navigate to https://www.merlinspotions.com/checkout/shipping/
- Observe you can navigate through checkout
- Please note there are no live links to this functionality.
